pilot/pkg/proxy/envoy/v2/mem.go

```diff
@@ -234,6 +234,7 @@ func (sd *MemServiceDiscovery) Services() ([]*model.Service, error) {
 	out := make([]*model.Service, 0, len(sd.services))
 	for _, service := range sd.services {
 		// Make a new service out of the existing one
+		// nolint: govet
 		newSvc := *service
```

Comment:
this is something very sketchy since the service struct has an inline mutex.
don't know what this is intentional or not.